pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -124,8 +124,10 @@ func (builder *podEntityDTOBuilder) BuildEntityDTOs(pods []*api.Pod) ([]*proto.E
 		if !controllable {
 			glog.V(3).Infof("Pod %v is not controllable.", displayName)
 		}
+		daemon := util.Daemon(pod)
 		entityDTOBuilder.ConsumerPolicy(&proto.EntityDTO_ConsumerPolicy{
 			Controllable: &controllable,
+			Daemon:       &daemon,
```

Comment:
I think one of the main motivation of not counting daemons as overhead in Turbo server is we still want to resize daemon (at least RECOMMEND), shall we mark them not controllable here?

Reply:
As discussed, the controllable and daemon settings are independent of each other, which allows for daemons to be resized.